lib/yao/resources/action.rb

```diff
@@ -0,0 +1,16 @@
+module Yao::Resources
+  module Action
+    def action(id, query)
+      res = POST(action_path(id)) do |req|
+        req.body = query.to_json
+        req.headers['Content-Type'] = 'application/json'
```

Comment:
Wow, I found `c.request :json` won't add any Content-Type headers:
https://github.com/lostisland/faraday_middleware/blob/master/lib/faraday_middleware/request/encode_json.rb

So this header should be explicit!
I understand.